src/Radio/index.js

```diff
@@ -96,13 +168,24 @@ Radio.propTypes = {
 
     className: propTypes.string,
     label: propTypes.string,
+    autocomplete: propTypes.string,
+    tabIndex: propTypes.string,
+
+    onFocus: propTypes.func,
+    onBlur: propTypes.func,
 
     required: propTypes.bool,
     checked: propTypes.bool,
     disabled: propTypes.bool,
     valid: propTypes.bool,
     warning: propTypes.bool,
     error: propTypes.bool,
+    autofocus: propTypes.bool,
+    readonly: propTypes.bool,
+}
+
+Radio.defaultProps = {
+    tabIndex: '0',
```

Comment:
Remove the default and let the browser handle it unless it is set explicitly.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/tabIndex